src/Illuminate/Events/CallQueuedListener.php

```diff
@@ -40,18 +40,18 @@ class CallQueuedListener implements ShouldQueue
     public $tries;
 
     /**
-     * The number of seconds to wait before retrying the job.
+     * The number of seconds before a released job will be available.
      *
      * @var int
      */
-    public $retryAfter;
+    public $backoff;
```

Comment:
Others have already commented about the nomenclature of `backoff` here.

I just want to add that `backoff` has a certain connotation of gradually backing off, so that each time it fails the delay is increased :man_shrugging:

Reply:
Hello 👋🏼 

I know no one asked for my opinion here, but what triggered me writing this comment is that, now that I was faced with the delay/backoff rename in a project and researched how this came about, I was under the same impression and thought "this doesn't feel right". As you said: "gradually backing off" was also my expectation

But, IMHO, turns out we're mostly just so used to the (indeed very common place) e.g. "exponential backoff" that it's easy to assume (even myself) the "backoff" equals "exponential backoff".

But in fact there are just different types of backoff one can do:
- linear
- constant
- exponential
- …

My 2c FWIW 😄